### ISSUE TITLE
Duplicate method ensureRipple in PaperIconButtonLight(Element)

### ISSUE THREAD

**Issue by muelbr** [vaadin/gwt-api-generator#67]
There is a duplicate method `ensureRipple` in the Widget and Element class because `paper-icon-button-light.html` defines `ensureRipple: function(var_args)`  and uses `paper-ripple-behavior.html` which defines `ensureRipple: function(optTriggeringEvent)`

**Comment by cpboyd** [comment]
A simple fix is to change [line 66 in gulpfile.js](https://github.com/vaadin/gwt-api-generator/blob/master/gulpfile.js#L66) to:
```js
// merge properties
        if (nestedBehaviors.properties && nestedBehaviors.properties.length) {
          nestedBehaviors.properties.forEach(function (property) {
            var notDuplicate = _.filter(item.properties, function (p) {
                return p.name == property.name;
              }).length === 0;
            if (notDuplicate) {
              item.properties.push(property);
            }
          });
        }
```

This, effectively, prevents this duplication from occurring.  Granted, it doesn't give you the access to the original behavior's method, but I'm not certain that'd be useful in this specific case.

**Comment by cpboyd** [comment]
I'd make a pull-request with this change, but I've still got a pending pull-request from several months ago of a relatively trivial change.

**Comment by manolo** [comment]
Fixed in bdc3b645d5a4f2b7063114060f093ad3a3185129
